dbt-tests-adapter/pyproject.toml

```diff
@@ -50,6 +42,9 @@ Changelog = "https://github.com/dbt-labs/dbt-adapters/blob/main/CHANGELOG.md"
 requires = ["hatchling"]
 build-backend = "hatchling.build"
 
+[tool.hatch.version]
```

Comment:
This got moved from troubleshooting, but it's position now aligns with the root `pyproject.toml`, so it feels right to keep it here.